feconf.py

```diff
@@ -635,6 +635,17 @@ def get_empty_ratings():
 # questions.
 MAX_NUMBER_OF_SKILL_IDS = 20
 
+# The maximum number of blog post cards to be visible on each page in blog
+# homepage.
+MAX_LIMIT_FOR_CARDS_ON_HOMEPAGE_PAGE = 10
```

Comment:
MAX_NUM_CARDS_TO_DISPLAY_ON_BLOG_HOMEPAGE
MAX_NUM_CARDS_TO_DISPLAY_ON_BLOG_AUTHOR_PAGE
MAX_NUM_CARDS_TO_DISPLAY_ON_BLOG_POST_PAGE

I don't understand the last one, actually. What is it?

Reply:
We have this at the end of the blog post page:
![Uploading Screenshot from 2021-07-04 23-36-35.png…]()
Link: https://xd.adobe.com/view/9bb82409-cdca-432a-b11c-88324643e2c0-ceeb/screen/24bd9402-b858-434f-9b5c-1fac4c21f9c7/
So, the last constant specifies the number of blog post cards to be displayed on this page.

Reply:
MAX_POSTS_TO_RECOMMEND_AT_END_OF_BLOG_POST, then. That captures the functionality of these cards better (otherwise no one really knows what cards you're talking about, and these cards seem a bit different in nature from the pages which naturally contain lists of cards).

Reply:
Done.

Reply:
The original comment in this thread does not seem to be fixed.

Reply:
Done.